pulsar-client/src/main/java/com/yahoo/pulsar/client/api/MessageId.java

```diff
@@ -44,4 +44,11 @@
     public static MessageId fromByteArray(byte[] data) throws IOException {
         return MessageIdImpl.fromByteArray(data);
     }
+    
+    /**
+     * Get messageId in string format
+     * 
+     * @return MessageId in string format
+     */
+    public String getMsgIdAsString(String entitySeparator);
```

Comment:
Are you suggesting that we change the toString() to return the new format? Won't it break anyone relying on old format?